src/credentials/FabricTable.cpp

```diff
@@ -2080,4 +2080,19 @@ CHIP_ERROR FabricTable::GetFabricLabel(FabricIndex fabricIndex, CharSpan & outFa
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR FabricTable::GetNextFabricAdditionIndex(FabricIndex * nextIndex)
```

Comment:
- Suggest "PeekFabricIndexForNextAddition"
- Rename nextIndex to `outIndex`
- Use a ref rather than a pointer --> avoids the null issue statically

Reply:
Done for all three.